fsw/cfe-core/src/inc/ccsds.h

```diff
@@ -169,7 +169,7 @@ typedef struct
      * In Version 1 mode, the standard / non-APID qualified header is used for all packets
      */
     CCSDS_PriHdr_t      Hdr;    /**< Complete "version 1" (standard) header */
-
+  
```

Comment:
Whitespace change?  Did you mean to do this?

Reply:
No.  I accidentally deleted the #ifdef MESSAGE_FORMAT_IS_CCSDS_VER_2 line when I originally went through the file...and must have added a white space when I replaced it.  I can go ahead and remove it.